src/ixdat/spectra.py

```diff
@@ -647,6 +667,7 @@ def spectrum_series(self):
         """The `SpectrumSeries` with the spectral data"""
         if isinstance(self._spectrum_series, PlaceHolderObject):
             self._spectrum_series = self._spectrum_series.get_object()
+        self._spectrum_series.tstamp = self.tstamp
```

Comment:
Why should the `tstamp` be reassigned every time this property is used?

Reply:
The `SpectrumSeries` returned here should have the same `tstamp` as the `Measurement`, so that the `Spectrum` objects it contains have `spectrum.t` with respect to the `Measurement.tstamp` so that anyone with a plot of the measurement can identify when the spectrum was taken. The `Measurement`'s tstamp can be reset by the user at any time by e.g. `meas.tstamp += 1000` to switch the absolute time at which relative t=0. 
The proper way to meet these requirements, I think, is to make `Measruement.tstamp` a property with a setter and a getter, the setter of `SpectroMeasurement` being modified to update `self._spectrum_series.tstamp`. I just liked `tstamp` being a simple attribute, so took this lazy solution.
I will fix it, doing it the "right" way just described instead.

Reply:
Done.
Turns out `tstamp` was already a protected property :) 
So very little to add.